puppet/modules/web/manifests/init.pp

```diff
@@ -129,6 +131,24 @@
     max_connections => $max_rsync_connections,
   }
 
+  # DOCS
+  $docs_directory = '/var/www/vhosts/docs/htdocs'
+  $docs_directory_config = [
+    {
+      path            => $docs_directory,
+      options         => ['Indexes', 'FollowSymLinks', 'MultiViews'],
+      expires_active  => 'on',
+      expires_default => 'access plus 4 hours',
```

Comment:
Is it wise to set caching headers? This depends on the build process and probably safer to start without.

Reply:
I copy-pasted from the yum example, do you want me to remove it?